easybuild/easyconfigs/o/ORCA/ORCA-3_0_2-linux_x86-64_openmpi_165.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PackedBinary"
+
+name = "ORCA"
+version = '3_0_2-linux_x86-64'
```

Comment:
you're missing a versionsuffix here, the easyconfig filename now doesn't match the contents, please fix (and try not to hardcode the OpenMPI version in the versionsuffix, only specify it once, see easyconfigs that have `-Python-2.7.8` as versionsuffix, for example)